gimel/logger.py

```diff
@@ -35,7 +35,6 @@ def setup(name=__name__, level=logging.INFO):
         return logger
     logger.setLevel(level)
     try:
-        import click
```

Comment:
I actually need this, so we can swap the formatter if click isn't available (which is what happens on AWS Lambda).